alacritty/src/main.rs

```diff
@@ -235,9 +235,13 @@ fn run(
 }
 
 fn log_config_path(config: &Config) {
+    if config.ui_config.config_paths.is_empty() {
+        return
+    }
+
     let mut msg = String::from("Configuration files loaded from:");
     for path in &config.ui_config.config_paths {
-        msg.push_str(&format!("\n  {:?}", path.display()));
+        msg.push_str(&format!("\n- {}", path.display()));
```

Comment:
Why did you change this?

Reply:
> 2. Print out the paths with Display in a YAML-like format.

It was mostly stylistic, sorry if there was some history here I'm stepping on.

Reply:
By stripping the debug option it becomes more difficult to see things like excessive whitespace, so that's no good.

And I'm also not a fan of the `-` at the beginning. I much prefer things actually being indented when they belong to the previous statement, since that makes it much simpler to skip over them if you don't care about it.

Reply:
That's a fair point about irritating file paths with whitespace (I had a feeling you might bring this up). The indentation is already pretty deep due to the way `info!` messages work, so that's actually part of why I changed this.

Reply:
I don't see any problem with that.